messageapp/models/Message.model.js

```diff
@@ -0,0 +1,21 @@
+const { Schema, model } = require("mongoose");
+
+const messageSchema = new Schema(
+    {
+        destination: {
+            type: String,
+            required: [true, "Destination is required"],
+            trim: true
```

Comment:
what does the `trim` field do?

Reply:
What trim does is to eliminate blanks at both ends of a String.